### ISSUE TITLE
Localization story

### ISSUE THREAD

**Issue by vors** [PowerShell/PowerShell#666]
Question came out during our onboarding hackathon: "Who will localize our resources when we move to github?"

Assigned to Joey to follow-up.

**Comment by joeyaiello** [comment]
Assigning this to me to follow-up with legal to determine to what extent we need this.

**Comment by iSazonov** [comment]
Any news?

**Comment by mklement0** [comment]
Copying over a simple Pester test for localized error messages from the now closed duplicate #10679:

```powershell
[cultureinfo]::CurrentUICulture = 'de-DE'; Get-Item nosuch 2>&1 | Should -match 'nicht gefunden'
```
